src/hooks/useTransfer.ts

```diff
@@ -35,45 +36,34 @@ export function useTransfer(selectUnit: Ref<string>, decimal: Ref<number>, fn?:
   const substrateAccounts = computed(() => store.getters['general/substrateAccounts']);
   const isEthWallet = computed(() => store.getters['general/isEthWallet']);
 
-  const transferLocal = async (transferAmt: BN, fromAddress: string, toAddress: string) => {
+  const transferNative = async (transferAmt: BN, fromAddress: string, toAddress: string) => {
```

Comment:
`useSendTx` is still not being used anywhere. I think you might want to remove unused functions.

Reply:
Fixed.
https://github.com/AstarNetwork/astar-apps/pull/378/commits/86856d94e3ed1591385c8707defb8e6539809509